packages/turbo-types/src/types/config.ts

```diff
@@ -192,11 +194,13 @@ export interface Pipeline {
   passThroughEnv?: null | Array<EnvWildcard>;
 
   /**
+   * Deprecated as of Turborepo 2.0.0. Consider using `inputs` instead.
```

Comment:
```suggestion
   * @deprecated as of Turborepo 2.0.0. Consider using {@link Pipeline.inputs} instead.
```

In addition to using @deprecated, you can use the @link tag to point to the new value to use, directly, which means people can just click it to be taken to that field:

![Screenshot_20240814_233841](https://github.com/user-attachments/assets/b3487201-066a-436b-bd45-afbb978c39b0)